README.md

```diff
@@ -34,6 +34,13 @@ If your project utilises Spring YAML you can define your Housekeeping within the
         # Database Password
         password: Ch4ll3ng3
 
+You can override spring boot data (HikariCP/hibernate) settings in the yaml by providing the relevant properties. Housekeeping defaults are added with lower precedence.
+For example override the default connection pool size to 5 add this:
+
+    spring.datasource.max-active: 5 
+    
+ See Spring boot documentation for a full list of properties that can be set.   
```

Comment:
purposely didn't add a link as we'll just forget to keep it up-to-date I think it's clear where to find this stuff